### DISCUSSION TITLE
Unable to import SortingState from @tanstack/react-table

### DISCUSSION THREAD

**Discussion by svb1998** [TanStack/table#5017] [Q&A] [answered]
Hi, I'm trying to import {SortingState} from @tanstack/react-table, but I get this error:

![image](https://github.com/TanStack/table/assets/61709741/431dbf39-e014-4077-ba28-390a9cc42719)

I'm using Reacct with Javascript, not Typescript and Vite.

**Comment by wojciech-space** [comment, marked as answer]
Hi @svb1998, sorry for being late to the party, I had the same issue but just found the solution, if you use TypeScript, you have to import `SortingState` as a type:

```
import type { SortingState } from "@tanstack/react-table";
```

**Reply by svb1998** [reply]
Thank you @wojciech-space, better late than never! Great to know it, I'll apply it in future projects 👍🏻
